### PR TITLE
Make sure tests are run when docker image is created

### PR DESCRIPTION
**Pull request by halkeye**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "incrementals-publisher",
-  "version": "1.2.0",
+  "version": "1.3.0",
```
Comment (jglick) [inline package-lock.json:3]: ?

**Review by jglick — APPROVED** [review]
Greek to me but sounds OK!

**Comment by jglick** [comment]
(Still no CI though I see.)

File: package-lock.json (hunk 1 above)
Comment (halkeye) [reply package-lock.json:3]: @timja released 1.3, but i guess manually so package-lock didn't get updated. when i did an npm install the lock file also updated.

**Comment by halkeye** [comment]
my jobdsl is weak, but I think we need to copy most of https://github.com/jenkins-infra/charts/blob/master/config/default/jenkins-infra.yaml#L284-L309 into https://github.com/jenkins-infra/charts/blob/master/config/default/jenkins-infra.yaml#L335

or rename the repo to docker-incrementals-publisher

File: package-lock.json (hunk 1 above)
Comment (timja) [reply package-lock.json:3]: I didn’t realise it was in there as well =/.

Comment (halkeye) [reply package-lock.json:3]: for node best way to bump version is `npm version $arg` but it honestly doesn't matter, i just felt it was worth doing

**Review by markyjackson-taulia — APPROVED** [review]

**Comment by jglick** [comment]
Oh this is all on infra.ci.jenkins.io? Then I cannot offer advice.

**Comment by halkeye** [comment]
I found https://ci.jenkins.io/job/Infra/job/incrementals-publisher/job/master/ which is better for PRs, while infra publishes.

So i refactored the Jenkinsfile so it can run on both.

**Comment by timja** [comment]
Something is wrong with the ci.jenkins.io app on the jenkins-infra org:

```
 Error while processing pull request 15
java.io.FileNotFoundException: https://api.github.com/repos/jenkins-infra/incrementals-publisher/collaborators/halkeye/permission
	at org.kohsuke.github.extras.okhttp3.ObsoleteUrlFactory$OkHttpURLConnection.getInputStream(ObsoleteUrlFactory.java:490)
	at 
```

cc @olblak

https://ci.jenkins.io/job/Infra/job/incrementals-publisher/indexing/console

**Comment by halkeye** [comment]
I wonder if the infra folder isn't using the app credentials? I don't have extended read on ci so can't check

**Comment by timja** [comment]
image

**Comment by halkeye** [comment]
\o/ CI passes
thanks @olblak

**Review by olblak — APPROVED** [review]

**Pull request merged**